site/content/docs/latest/howto/access-control.md

```diff
@@ -1,30 +1,52 @@
 # Access Control in Kubeapps
 
-Before you configure the Kubernetes Role-Based-Access-Control for your authenticated users to access Kubeapps you will first need to establish how users will authenticate with the Kubernetes clusters on which Kubeapps operates.
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [User Authentication in Kubeapps](#user-authentication-in-kubeapps)
+   1. [OpenID Connect authentication](#openid-connect-authentication)
+   2. [Service Account authentication](#service-account-authentication)
+3. [Assigning Kubeapps User Roles](#assigning-kubeapps-user-roles)
+   1. [Applications](#applications)
+      1. [Read access to Applications within a namespace](#read-access-to-applications-within-a-namespace) 2.[Write access to Applications within a namespace](#write-access-to-applications-within-a-namespace)
```

Comment:
```suggestion
      1. [Read access to Applications within a namespace](#read-access-to-applications-within-a-namespace)
      2.[Write access to Applications within a namespace](#write-access-to-applications-within-a-namespace)
```

Reply:
I have used the `format document` option in Visual Studio before saving it, so this could change that. Thanks!